composer.json

```diff
@@ -1,20 +1,20 @@
 {
     "config": {
         "platform": {
-            "php": "7.0.8"
+            "php": "7.1"
```

Comment:
The min core version for `user_ldap` is already set to 10.4, and that requires at least PHP 7.1. So this change does not effect which core ownCloud10 that user_ldap will install on.